indico/modules/rb/controllers/user/reservations.py

```diff
@@ -461,6 +461,19 @@ def _make_form(self):
         self.past_date = self.date_changed = False
         changes = {'room_id': self._room.id}
 
+        # Move the start date to the current day, adjust the end date
```

Comment:
If we need this massive comment I would move the logic to a helper function and put this as a docstring.